package.json

```diff
@@ -43,28 +43,73 @@
     "webdriverjs"
   ],
   "scripts": {
-    "test": "grunt test",
-    "build": "grunt build"
+    "lint": "eslint './lib/**/*.js' './test/**/*.js' --fix",
+    "fmt": "prettier --write './lib/**/*.js' './test/**/*.js'",
+    "precommit": "lint-staged",
+    "test": "mocha test/unit/*.js test/integration/*.js",
```

Comment:
Are you avoiding reusing the existing scripts (`test: npm run test:unit && npm run test:integration`) because it would have to run mocha twice? I'm wondering because, as-is, there's a small chance that `npm run test` and `npm run test:unit` will run different unit tests if one is changed without changing the other.

Reply:
No real reasoning here. I see what you mean and will update this.